### ISSUE TITLE
[Accessibility]: va-accordion and va-additional-info - "collapsed" and "expanded" not announced on iOS VoiceOver

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/vets-design-system-documentation#5216]
### Point of contact

Jamie Klenetsky Fay

### How severe do you think this issue is?

Critical

### Severity rationale

This could potentially fall under 2 level 1 categories:
- Content being added or removed from the page must announce a change to assistive devices.
- Improper aria attributes or missing IDs must be corrected.

### URL where issue occurs

https://design.va.gov/storybook/?path=/story/uswds-va-accordion--default

### Component or pattern (if applicable)

va-accordion, va-additional-info

### Environment details

- OS: iOS 26
- Browser: Safari
- AT: VoiceOver

### Steps to reproduce

1. Put focus on "Expand all" button, let the announcement finish
2. Double tap - the "expanded" announcement is cut off"
3. Do the same, but for "Collapse all"
4. Do the same, but for the "Additional Information" button on [this page](https://design.va.gov/storybook/?path=/story/uswds-va-additional-info--default)

### Observed behavior

VoiceOver appears to start announcing "expanded" or "collapsed," but it's immediately cut off and doesn't get announced fully. 

Because of this, users wouldn't know if they've actually expanded/collapsed the button or not.

### Expected behavior / Acceptance criteria

VoiceOver should announce "expanded" when expanded and "collapsed" when collapsed

### Supporting evidence (screenshots, recordings, code snippets)

https://github.com/user-attachments/assets/cc2ec505-17ef-48e0-979e-23c651b61535

https://github.com/user-attachments/assets/e60d9510-ad42-48f2-80e3-98ffe390ba79

### Type of issue

- [ ] Axe-core
- [ ] Cognitive | user flows & navigating
- [ ] Color, typography, & visual elements
- [x] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [ ] Focus
- [ ] Headings
- [ ] Keyboard
- [ ] Link and button labels
- [x] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [x] Screenreader | assistive tech & device support
- [x] Semantic HTML
- [ ] URL standards
- [ ] Zoom or Magnification
- [ ] Something else
